src/Backend/OnboardingInsuranceAPI/Program.cs

```diff
@@ -23,7 +27,8 @@ static async Task Main()
             {
                 services
                     .AddScopedByInterface<IHandler>()
-                    .AddDbContext<DataContext>(); //AddDbContextFactory try booth
+                    .AddCosmosDb(); //Dev DB
+                    //.AddInMemoryDb(); //Test DB
```

Comment:
X

Reply:
Why comment it out? You will never use it here anyway.

Reply:
I wanted to test how its will work InMemory, but yeah I'll delete this.